src/main/java/tech/fastj/systems/audio/AudioEventListener.java

```diff
@@ -6,6 +6,8 @@
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 
+import static tech.fastj.systems.audio.state.PlaybackState.Stopped;
```

Comment:
Is there any part of the code which requires this import? I might be missing it from my mobile, but just to be sure.

Reply:
My editor doesn't seem too happy with `Stopped` and `Paused` as it can't find a path to them.

Reply:
Interesting... I'll have a look into the source code right now.

Reply:
After a bit of inspection, I agree -- that's a very weird bit of code you've found! I think IntellIJ on my side sees no issues with it because it resolves the Paused and Stopped as possibilities from the `getPlaybackState()` method call. It doesn't make too much sense that an import isn't required, given how Java normally is, but it is what it is I suppose.
What editor are you using that's giving you this issue?

Reply:
Funny enough, I'm using IntelliJ too!

Reply:
The mysteries of technology 🤣 
Either way, go ahead and remove the import. Might be worth looking into that import issue some more, but for this repo everything is fine.